project/plugins/aws/__init__.py

```diff
@@ -91,20 +103,21 @@ def removeUser(email, configMap,allPermissions, plugin_tag):
         aws_secret_access_key=Secret
     )
 
-    log = plugin_tag + ': ' + username + ' removed from organization.\n'
-    instruction = email.split('@', 1)[0] + removalMessage(configMap, plugin_tag)
-
     try:
         # remove from groups
         response = client.list_groups_for_user(UserName=username)
         groups = response.get('Groups')
 
     except client.exceptions.NoSuchEntityException:
-        print(plugin_tag + ' error: ' + username + ' could not be deleted, because it does not exist')
+        log = (plugin_tag + ' error: ' + username + ' could not be deleted, because it does not exist')
+        instruction = log
```

Comment:
Why do we need to set the instruction to the value of log?  Could we not just use log the whole time?